src/services/ceramic-service.ts

```diff
@@ -40,4 +41,21 @@ export default class CeramicServiceImpl implements CeramicService {
 
     return (await Promise.race([streamPromise, timeoutPromise])) as T
   }
+
+  async multiQuery(queries: MultiQuery[]): Promise<Record<string, Stream>> {
+    let timeout: any;
+
+    const queryPromise = this._client.multiQuery(queries)
+      .finally(() => {
+        clearTimeout(timeout);
+      });
+
+    const timeoutPromise = new Promise<Record<string, Stream>>((_, reject) => {
+      timeout = setTimeout(() => {
+        reject(new Error(`Timed out loading multiquery`))
+      }, 60 * 1000);
```

Comment:
Is 60 seconds too much?

Reply:
I don't think so.  And with streams being loaded in concurrent batches of 128 streams at a time, one stream taking a long time to time out doesn't actually tie up the rest of the system very much.